src/plugins/teleport-plugin-vue-base-component/utils.ts

```diff
@@ -84,7 +84,7 @@ export const generateVueNodesTree = (
           const alternateNode = getNodeFromContent(alternateContent, accumulators)
           htmlUtils.addAttributeToNode(consequentNode, 'v-if', conditionalStatement)
           htmlUtils.addChildNode(htmlNode, consequentNode)
-          htmlUtils.addBooleanAttributeToNode(alternateNode, 'v-else')
```

Comment:
I propose a different solution to this, please keep `addBooleanAttributeToNode` and modify that function from `project-utils` to add the empty string instead of adding `true`. this way, you solve the problem in all scenarios and we keep the specialized function for adding boolean attributes

Reply:
Thought the same thing @alexnm but if we see here ``addBooleanAttributeToNode`` works as the same for a list of predefined HTML attributes reference --> https://github.com/Prettyhtml/prettyhtml/blob/master/packages/prettyhtml-hast-to-html/lib/element.js#L234 by the lib itself, only for the attributes which are not predefined behave in this way.

How about writing a new method and to add empty strings to use in these cases like ``addOverloadBooleanAttributeToNode``? and for all the usual HTML attributes we can use ``addBooleanAttributeToNode``

Reply:
can we import that list of attributes and do a conditional inside our util function? the goal is to provide an API with no friction and no need to know the details about pretty html and which boolean values are allowed

Reply:
Yeah @alexnm that will give a smoother experience but if we see here --> https://github.com/Prettyhtml/prettyhtml/blob/master/packages/prettyhtml-hast-to-html/lib/element.js#L225 again they were using another dependency to find out which notes they support, so every time we try to add value to the node they do a cross-check here and then add the value to attribute 😅 

Even if we find the list, then there will check happening two times, one on our side and again on their side 😅  .

Reply:
But still, smother experience comes first, I will find something for this today and update here 👍

Reply:
Or we can just change our ``addBooleanAttributeToNode`` to add empty string by default so it just wants we needed, here is how they are handeling --> https://github.com/Prettyhtml/prettyhtml/blob/master/packages/prettyhtml-hast-to-html/lib/element.js#L265 so, they would do the two jobs we need.

1. Smoth Dev Experience
2. Just add the attributenames when we want to add a boolean state

Something like this

~~~
export const addBooleanAttributeToNode = (node: HastNode, key: string) => {
  node.properties[key] =''
}
~~~

If we are ok with this, i will make the changes and update the branch 👍

Reply:
go with the empty string please, and leave a comment explaining why `true` doesn't always work